optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/plugin/routing/AirDistanceRouter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.vehiclerouting.plugin.routing;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+
+import org.optaweb.vehiclerouting.domain.Coordinates;
+import org.optaweb.vehiclerouting.service.distance.DistanceCalculator;
+import org.optaweb.vehiclerouting.service.region.BoundingBox;
+import org.optaweb.vehiclerouting.service.region.Region;
+import org.optaweb.vehiclerouting.service.route.Router;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConditionalOnProperty(prefix = "app.routing", name = "engine", havingValue = "air")
+public class AirDistanceRouter implements Router, DistanceCalculator, Region {
```

Comment:
Depending if you value accuracy over speed, you might want to use Great-circle distance: https://en.wikipedia.org/wiki/Great-circle_distance; your given formula works well enough, although paths with different longitudes might be inaccurate.

Reply:
Good idea. This was a quick-and-dirty approximation. I'll replace it with https://en.wikipedia.org/wiki/Haversine_formula in the next PR. If we allow the user to configure the travel speed this might actually become a useful feature.

Reply:
#11